### PR TITLE
Update the missed out English names in English and Hindi TSV files

### PR DESCRIPTION
**Pull request by bikashtudu**
|PR checklist|Status| 
|-----|:------:| 
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Pehla-School/issues/321)| 
|Are the changes replicated across different locales?|NA| 
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA| 
|Is code analysis task executed and issues resolved in the CPP/native code?|NA| 
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA| 
|Is code commenting done properly for the changes made?|NA| 
|Is correct variable naming convention followed?|NA| 
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA| 
|Are the message strings used in the application taken from strings.xml file?|NA| 
|Are the permissions which are not required removed from AndroidManifest.java file?|NA| 
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA| 
|Are the generated files correctly configured in .gitignore?|NA| 
|Are the release signing configurations removed from the gradle files?|NA| 
|Is commented code removed from all files?|NA| 
|Are updated images compressed (< 100 KB)?|NA| 
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|NA|

### CODE REVIEW

File: localized/games/eggquiz/eggquizliteracy_levels_en.tsv [hunk 1]
```diff
@@ -637,10 +637,10 @@ PostTest_9	1	9001, 9002, 9003, 9004, 9005, 9006, 9007, 9008, 9009, 9010
...
-			9007	imageseq_sentence		I am mad. I had a bad day at school.		Your day wasn't all bad.	I am mad. I had a bad day at school. Miss Nitika helped me up. Your day wasn't all bad.	“Miss Nitika helped me up.”	“I tripped over it.”^“Miss Nitika helped me up.”^“This boy laughed at me.”
+			9007	imageseq_sentence		I am mad. I had a bad day at school.		Your day wasn't all bad.	I am mad. I had a bad day at school. Miss Divya helped me up. Your day wasn't all bad.	“Miss Divya helped me up.”	“I tripped over it.”^“Miss Divya helped me up.”^“This boy laughed at me.”
 			9008	imageseq_sentence		book_en_413_page_4.png		book_en_413_page_8.png	We can bring melons to eat on the moon. We will eat them with spoon. Ajay said to June. Ajay and June built rockets boots with some tools. They ate their breakfast of melon with the spoon.	Ajay and June build rockets boots with some tools.	Ajay and June build rockets boots with some tools.^Ajay and June knew they must be home soon.^They landed back at home around noon.
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_en.tsv]: Update the word `June`. Change the word in the entire TSV

File: localized/games/eggquiz/eggquizmath_levels_hi.tsv [hunk 2]
```diff
@@ -346,15 +346,15 @@ SpecialTest_15	1	150001~150010, 150001~150010, 150001~150010, 150001~150010, 150
 			150006	word_problem	अलका के पास ९ गुब्बारे हैं। ६ ग्रीन  हैं और बाकी ब्लू हैं। ब्लू गुब्बारे कितने हैं??		अलका के पास ९ गुब्बारे हैं। ६ ग्रीन  हैं और बाकी ब्लू हैं। ब्लू गुब्बारे कितने हैं??				3
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_hi.tsv:346]: Change `सुश्री डॉसन` to Hindi name on  Line 344

File: localized/games/eggquiz/eggquizmath_levels_hi.tsv [hunk 3]
```diff
@@ -327,7 +327,7 @@ SpecialTest_13	1	130001~130010, 130001~130010, 130001~130010, 130001~130010, 130
 			130009	word_problem	समीरा एक जन्मदिन की पार्टी के लिए तैयारी कर रही है। उसने आज सुबह ६ गुब्बारे फुलाए और दुपहर को ५ गुब्बारे फुलाए। उसने सभी में कितने गुब्बारे फुलाए?	समीरा एक जन्मदिन की पार्टी के लिए तैयारी कर रही है। उसने आज सुबह ६ गुब्बारे फुलाए और दुपहर को ५ गुब्बारे फुलाए। उसने सभी में कितने गुब्बारे फुलाए?				11
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_hi.tsv:327]: Replace Chanakya text with Hindi on Line 186

File: localized/games/eggquiz/eggquizmath_levels_hi.tsv [hunk 4]
```diff
@@ -346,15 +346,15 @@ SpecialTest_15	1	150001~150010, 150001~150010, 150001~150010, 150001~150010, 150
 			150006	word_problem	अलका के पास ९ गुब्बारे हैं। ६ ग्रीन  हैं और बाकी ब्लू हैं। ब्लू गुब्बारे कितने हैं??		अलका के पास ९ गुब्बारे हैं। ६ ग्रीन  हैं और बाकी ब्लू हैं। ब्लू गुब्बारे कितने हैं??				3				
 			150007	word_problem	जिया और उसके भाई ने ६ आमों को उठाया। जिया ने ४ आम उठाए। उसके भाई ने कितने आम उठाए?	जिया और उसके भाई ने ६ आमों को उठाया। जिया ने ४ आम उठाए। उसके भाई ने कितने आम उठाए?				2				
 			150008	word_problem	सुमित और रेवेन ने एक साथ ८ तस्वीरें खींचीं। सुमित ने ३ तस्वीरें खींचे। रेवेन ने कितने तस्वीरें खींचे?	सुमित और रेवेन ने एक साथ ८ तस्वीरें खींचीं। सुमित ने ३ तस्वीरें खींचे। रेवेन ने कितने तस्वीरें खींचे?				5
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_hi.tsv]: Change `रेवेन` to Hindi Indian name. Replace the word wherever referenced

File: localized/games/eggquiz/eggquizmath_levels_hi.tsv [hunk 5]
```diff
@@ -405,9 +405,9 @@ SpecialTest_23	1	230001~230010, 230001~230010, 230001~230010, 230001~230010, 230
 			230004	word_problem	एक कक्षा में १३ लड़कियां और १८ लडक़े हैं। लड़कियों की तुलना में कितने अधिक लडक़े हैं?	एक कक्षा में १३ लड़कियां और १८ लडक़े हैं। लड़कियों की तुलना में कितने अधिक लडक़े हैं?				5
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_hi.tsv:405]: Replace `लुसी` to Hindi Indian name on Line 402

File: localized/games/wordwindow/wordwindow_level_hi.tsv [hunk 6]
```diff
@@ -32,15 +32,15 @@ en-US	2	3	5	4	"पुस्तकालय में स्कूल कार
...
 en-US	3	3	3	2	"पूजा के 9 कंगन हैं। वह उनमें से 2 खो देती है। पूजा के पास अभ कितने कंगन बचे हैं?"	पूजा के 9 कंगन हैं। वह उनमें से 2 खो देती है। पूजा के पास अभ कितने कंगन बचे हैं?	9	7	5	3	bracelets	bracelets	bracelets	bracelets	9 - 2 = <7>	2	9	2	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
 en-US	3	3	4	2	"आसमान में 9 पतंगें उड़ रही हैं। एक हवा 3 पतंगों को जमीन पर गिरा देती है। कितनी पतंगें अभी भी आसमान में उड़ रही हैं?"	आसमान में 9 पतंगें उड़ रही हैं। एक हवा 3 पतंगों को जमीन पर गिरा देती है। कितनी पतंगें अभी भी आसमान में उड़ रही हैं?	5	6	7	8	kites	kites	kites	kites	9 - 3 = <6>	2	9	3	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
 en-US	3	3	5	2	"पुष्प के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बची हैं?"	पुष्प के पास 7 बिस्कुट हैं। वह उनमें से 2 खाती है। उसके पास कितनी बिस्कुट बची हैं?	3	4	5	6	biscuits	biscuits	biscuits	biscuits	7 - 2 = <5>	3	7	2	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_hi.tsv:46]: Update `लुसी` on Line 49

File: localized/games/wordwindow/wordwindow_level_en.tsv [hunk 7]
```diff
@@ -73,11 +73,11 @@ en-US	6	1	1	4	"Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How m
...
-en-US	6	1	5	4	"There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?"	There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	11	14	17	20	chairs	chairs	chairs	chairs	14 + 3 = <17>	3	14	17	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
+en-US	6	1	5	4	"There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?"	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	11	14	17	20	chairs	chairs	chairs	chairs	14 + 3 = <17>	3	14	17	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_en.tsv:76]: Check if TTS works correctly for `Mr.`. Also, check if all the references of `Ms.` is replaced by `Miss`

File: localized/games/wordwindow/wordwindow_level_en.tsv [hunk 8]
```diff
@@ -194,11 +194,11 @@ en-US	14	3	2	4	"Namita has 13 flowers in her garden. Gauri has 8 flowers. How ma
...
 en-US	15	1	4	5	"Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?"	Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?	6	9	12	14	fireflies	fireflies	fireflies	fireflies	14 = 5 + <9>	2	5	9	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
-en-US	15	1	5	5	"Ms. Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?"	Miss Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
+en-US	15	1	5	5	"Miss Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?"	Miss Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_en.tsv]: Replace `Jagat` with a Female name

Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_en.tsv]: Remove `@` from the TTS text

File: localized/games/eggquiz/eggquizmath_levels_hi.tsv [hunk 9]
```diff
@@ -341,20 +341,20 @@ SpecialTest_15	1	150001~150010, 150001~150010, 150001~150010, 150001~150010, 150
 			150001	word_problem	अदिति के पास ५ किताबें हैं। उनमें से ३ विज्ञान की किताबें हैं और बाकी स्टोरीबुक हैं। स्टोरीबुक कितने हैं??	अदिति के पास ५ किताबें हैं। उनमें से ३ विज्ञान की किताबें हैं और बाकी स्टोरीबुक हैं। स्टोरीबुक कितने हैं??				2				
 			150002	word_problem	अमन की बिल्ली के ९ बच्चे थे। उनमें से ५ वाइट हैं और बाकी ग्रे हैं। बिल्ली के कितने ग्रे बच्चे थे?	अमन की बिल्ली के ९ बच्चे थे। उनमें से ५ वाइट हैं और बाकी ग्रे हैं। बिल्ली के कितने ग्रे बच्चे थे?				4				
 			150003	word_problem	एक कक्षा में ९ बच्चे हैं। उनमें से ६ लडक़े हैं। कक्षा में कितनी लड़कियाँ हैं?	एक कक्षा में ९ बच्चे हैं। उनमें से ६ लडक़े हैं। कक्षा में कितनी लड़कियाँ हैं?				3				
-			150004	word_problem	सुश्री डॉसन की कक्षा में १२ छात्र हैं। कुछ छात्र अनुपस्थित हैं और ७ छात्र उपस्थित हैं। कितने छात्र अनुपस्थित हैं?	सुश्री डॉसन की कक्षा में १२ छात्र हैं। कुछ छात्र अनुपस्थित हैं और ७ छात्र उपस्थित हैं। कितने छात्र अनुपस्थित हैं?				5				
+			150004	word_problem	सुश्री जगत की कक्षा में १२ छात्र हैं। कुछ छात्र अनुपस्थित हैं और ७ छात्र उपस्थित हैं। कितने छात्र अनुपस्थित हैं?	सुश्री जगत की कक्षा में १२ छात्र हैं। कुछ छात्र अनुपस्थित हैं और ७ छात्र उपस्थित हैं। कितने छात्र अनुपस्थित हैं?				5
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_hi.tsv]: Replace `जगत` with Female name

File: localized/games/eggquiz/eggquizmath_levels_en.tsv [hunk 10]
```diff
@@ -333,15 +333,15 @@ SpecialTest_14	1	140001~140010, 140001~140010, 140001~140010, 140001~140010, 140
...
 			140005	word_problem	Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How many stamps does Daanish have?	Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How many stamps does Daanish have?				8				
 			140006	word_problem	Jyoti is 5 years old. Her sister Jiya is 2 years older than Jyoti. How old is Jiya?	Jyoti is 5 years old. Her sister Jiya is 2 years older than Jyoti. How old is Jiya?				7				
-			140007	word_problem	There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?				17				
+			140007	word_problem	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?				17
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizmath_levels_en.tsv:336]: Check for `Mr.`

File: localized/games/wordwindow/wordwindow_level_en.tsv (hunk 7 above)
Comment (bikashtudu) [reply localized/games/wordwindow/wordwindow_level_en.tsv:76]: Checked Mr. is spoken correctly by the TTS, all the Ms. are replaced by Miss

File: localized/games/eggquiz/eggquizmath_levels_en.tsv (hunk 10 above)
Comment (bikashtudu) [reply localized/games/eggquiz/eggquizmath_levels_en.tsv:336]: Checked in the TTS, Mr. is spoken correctly by the TTS

File: localized/games/eggquiz/eggquizliteracy_levels_en.tsv [hunk 11]
```diff
@@ -637,10 +637,10 @@ PostTest_9	1	9001, 9002, 9003, 9004, 9005, 9006, 9007, 9008, 9009, 9010
...
+			9010	paragraph_sentence	Why did Lion summon Elephant to his log?	In the evening, Elephant was thirsty. She went back to the watering hole. When she tried to get a drink, several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large, he said. You must be careful of the frog because they are very small.			In the evening. Elephant was thirsty. She went back to the watering hole. When she tried to get a drink. several frogs got in her way. She accidentally stepped on two of them. Lion summoned Elephant to his log. You are very large. he said. You must be careful of the frogs. because they are very small.	To tell him to be careful.	To tell him to get a drink.^To tell him to leave the savanna.^To tell him to be careful.
```
Comment (Shailin) [inline localized/games/eggquiz/eggquizliteracy_levels_en.tsv:643]: Check if tab or space is added in this line

Comment (bikashtudu) [reply localized/games/eggquiz/eggquizliteracy_levels_en.tsv:643]: No tab or space is added, tested the updated TSV in English build

**Review by Shailin — DISMISSED** [review]

File: localized/games/wordwindow/wordwindow_level_en.tsv [hunk 12]
```diff
@@ -194,11 +194,11 @@ en-US	14	3	2	4	"Namita has 13 flowers in her garden. Gauri has 8 flowers. How ma
...
 en-US	15	1	4	5	"Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?"	Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?	6	9	12	14	fireflies	fireflies	fireflies	fireflies	14 = 5 + <9>	2	5	9	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
-en-US	15	1	5	5	"Ms. Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?"	Miss Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
+en-US	15	1	5	5	"Miss Payal has 12 students in her class. Some students are absent and 7 students are present. How many students are absent?"	Miss Payal has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png
```
Comment (Shailin) [inline localized/games/wordwindow/wordwindow_level_en.tsv]: Remove `@` from the TTS text. Have you checked how the TTS is coming with the `@` symbol?
Remove `@` from all the text of the TSV file

Comment (bikashtudu) [reply localized/games/wordwindow/wordwindow_level_en.tsv]: Removed @ from the TSV file

**Review by Shailin — DISMISSED** [review]

File: localized/games/wordwindow/wordwindow_level_hi.tsv [hunk 13]
```diff
@@ -92,16 +92,16 @@ en-US	7	2	5	2	"एक किसान ने 16 अंडे इकट्ठे
...
 en-US	8	3	4	5	"उर्मि की अलमारी में 7 शर्ट हैं। उनमें से 3 नीले हैं और बाकी हरे हैं। उसके पास कितनी हरी शर्ट है?"	उर्मि की अलमारी में 7 शर्ट हैं। उनमें से 3 नीले हैं और बाकी हरे हैं। उसके पास कितनी हरी शर्ट है?	3	4	5	6	green shirts	green shirts	green shirts	green shirts	7 = 3 + <4>	2	3	4	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
 en-US	8	3	5	5	"सुश्री अंजलि ने चीनी के 8 बैग का आदेश दिया। 5 बैग आज वितरित किए गए हैं। बाकी कल पहुंचाए जाएंगे। कल चीनी के कितने बैग वितरित किए जाएंगे?"	सुश्री अंजलि ने चीनी के 8 बैग का आदेश दिया। 5 बैग आज वितरित किए गए हैं। बाकी कल पहुंचाए जाएंगे। कल चीनी के कितने बैग वितरित किए जाएंगे?	2	3	5	8	bags	bags	bags	bags	8 = 5 + <3>	2	5	3	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png		
 en-US	9	1	1	6	"नकुल के पास 4 मार्बल्स हैं। सू ने नकुल को कुछ और मार्बल्स दिए और अब उसके पास कुल 6 मार्बल्स हैं। सू ने नकुल को कितने मार्बल्स दिए?"	ीटर के पास 4 मार्बल्स हैं। सू ने नकुल को कुछ और मार्बल्स दिए और अब उसके पास कुल 6 मार्बल्स हैं। सू ने नकुल को कितने मार्बल्स दिए?	2	3	4	5	marbles	marbles	marbles	marbles	4 + <2>= 6	1					4	2	+	B	wordwindow-stone-red.png	wordwindow-stone-blue.png
```
Comment (ankitk20) [inline localized/games/wordwindow/wordwindow_level_hi.tsv]: Fix issue with Hindi word on line 107. Also, search for other instances of similar issue in entire file using the character.

**Review by ankitk20 — CHANGES_REQUESTED** [review]

**Review by ankitk20 — APPROVED** [review]

**Pull request merged**